goto/the_real_goto.py

```diff
@@ -10,18 +10,17 @@
 from .gotomagic import text
 from .gotomagic.magic import GotoMagic
 from .gotomagic.utils import healthcheck
-
 from . import commands
 
-# make sure we print in utf-8
-try:
-    if sys.stdout.encoding != 'utf-8':
-        sys.stdout = codecs.getwriter('utf-8')(sys.stdout.buffer, 'strict')
-except:
-    pass  # TODO: implement utf-8 encoding of py2.7
-
 
 def main():
+    # make sure we print in utf-8
```

Comment:
oh I see why - it was moved. not removed.
Makes sense.

Would be cleaner to have it in it's own function somwhere I guess. -- but that is outside the scope of this PR - I'll fix it in the unicode branch